pyiron_atomistics/atomistics/structure/structurestorage.py

```diff
@@ -175,6 +178,16 @@ def add_structure(self, structure, identifier=None, **arrays):
             **kwargs: additional arrays to store for structure
         """
 
+        # Extract energies, forces, and atomic charges from ASE Atoms objects.
+        if isinstance(structure, ASEAtoms):
```

Comment:
```suggestion
        if isinstance(structure, ASEAtoms) and not isinstance(structure, Atoms):
```
otherwise this will be true for our `Atoms` as well.